### PR TITLE
docs ActionConfirmWidget, ActionCreateTiddlerWidget best practice examples

### PR DESCRIPTION
**Pull request by pmario**
docs ActionConfirmWidget, ActionCreateTiddlerWidget best practice examples

As discussed at: **[BUG] Doc examples should use best-practice for: button- and action-widgets** #7443

This PR contains changes for 2 widgets ActionConfirmWidget and ActionCreateTiddlerWidget  to make the example code easier to maintain. 

* Every example works if the buttons are clicked in the "renders as" section and
* if copy & pasted into a new tiddler, the example text still makes sense and works.

### CODE REVIEW

File: editions/tw5.com/tiddlers/widgets/ActionCreateTiddlerWidget.tid [hunk 1]
```diff
@@ -20,8 +20,8 @@ The ''action-createtiddler'' widget is invisible.
 
 |!Attribute |!Description |
 |$basetitle |The initial title that will be attempted. If a tiddler with that title already exists, then a numerical counter is added to the title and incremented until it is unique|
-|$savetitle |<<.deprecated-since "5.1.20" "ActionCreateTiddlerWidget Example 5">> A text reference identifying a field or index into which the title of the newly created tiddler will be stored after it is created |
```
Comment (Jermolene) [inline editions/tw5.com/tiddlers/widgets/ActionCreateTiddlerWidget.tid:23]: Why the change in the deprecation cutoff version?

File: editions/tw5.com/tiddlers/widgets/ActionCreateTiddlerWidget.tid [hunk 2]
```diff
@@ -32,42 +32,131 @@ The ''action-createtiddler'' widget is invisible.
 <<.from-version "5.2.0">> The content of the <<.wid "action-createtiddler">> widget is executed after the new tiddler has been created. The title of the newly created tiddler is stored in the variable <<.var "createTiddler-title">>.
 
 |!Variables |!Description |
-|`createTiddler-title` |The tittle of the tiddler that has been created. See [[ActionCreateTiddlerWidget Example 5]] |
+|`createTiddler-title` |The tittle of the tiddler that has been created. See the 5th example |
```
Comment (Jermolene) [inline editions/tw5.com/tiddlers/widgets/ActionCreateTiddlerWidget.tid:35]: Why replace the link with the clumsy textual reference?

File: editions/tw5.com/tiddlers/widgets/ActionCreateTiddlerWidget.tid [hunk 3]
```diff
@@ -32,42 +32,131 @@ The ''action-createtiddler'' widget is invisible.
...
 |`createTiddler-draftTitle` |This variable only exists to have feature parity with the deprecated parameters. It contains the title of a "draft tiddler" |
 
 ! Examples
 
 <<<
-<$macrocall $name='wikitext-example-without-html' src={{ActionCreateTiddlerWidget Example}}/>
+1) This example will create a ''new tiddler'' every time the button is clicked.
```
Comment (Jermolene) [inline editions/tw5.com/tiddlers/widgets/ActionCreateTiddlerWidget.tid:41]: Using hardcoded list numbering is not a practice I would want to recommend. If the worst comes to the worst, it would be better to use HTML syntax for `<ol>/<li>`.

**Review by Jermolene — COMMENTED** [review]
Thanks @pmario. I'm not sure that some of these changes are improvements. Why have you got rid of the separate example tiddlers? It seems more useful to be able to link to them than asking the user to count their way to them.

**Review by Jermolene — COMMENTED** [review]
Thanks @pmario. I'm not sure that some of these changes are improvements. Why have you got rid of the separate example tiddlers? It seems more useful to be able to link to them than asking the user to count their way to them.

File: editions/tw5.com/tiddlers/widgets/ActionCreateTiddlerWidget.tid (hunk 1 above)
Comment (pmario) [reply editions/tw5.com/tiddlers/widgets/ActionCreateTiddlerWidget.tid:23]: See: https://tiddlywiki.com/archive/full/TiddlyWiki-5.1.20.html#ActionCreateTiddlerWidget `$savedrafttitle` was new in 5.1.20
See: https://tiddlywiki.com/archive/full/TiddlyWiki-5.1.23.html#ActionCreateTiddlerWidget where both settings where deprecated. 

It probably was a "copy paste" error using the version number from `$savedrafttitle` as deprecation version, which is wrong.

File: editions/tw5.com/tiddlers/widgets/ActionCreateTiddlerWidget.tid (hunk 2 above)
Comment (pmario) [reply editions/tw5.com/tiddlers/widgets/ActionCreateTiddlerWidget.tid:35]: Usually action-widget doc tiddlers contain their examples in the doc tiddler. ... That's why I did remove them for consistency reasons.

File: editions/tw5.com/tiddlers/widgets/ActionCreateTiddlerWidget.tid (hunk 3 above)
Comment (pmario) [reply editions/tw5.com/tiddlers/widgets/ActionCreateTiddlerWidget.tid:41]: With examples we do have a lot of hardcoded numbering. eg: https://github.com/Jermolene/TiddlyWiki5/pull/7426/files#diff-ae90cbf3713502ed3ebecc9f345a9e5c95dbb06986e618697daa323025611ead from some docs recently merged.

**Comment by pmario** [comment]
>... Why have you got rid of the separate example tiddlers?

As I wrote ... For consistency with all other widget docs.

> It seems more useful to be able to link to them than asking the user to count their way to them.

I did implement the example tiddlers some time ago fro the wrong reason. I thought it would not be possible to have `wikitext-example-without-html` with `\define macro()` in it. ... I was wrong there. 

-------

I also don't like the search results showing the examples. It looks messy.

image

File: editions/tw5.com/tiddlers/widgets/ActionCreateTiddlerWidget.tid (hunk 2 above)
Comment (pmario) [reply editions/tw5.com/tiddlers/widgets/ActionCreateTiddlerWidget.tid:35]: Also the numbering does not work well, if the order should be changed. If we do have 1 tiddler per example they should have better names -- other than 1, 2, 3

File: editions/tw5.com/tiddlers/widgets/ActionCreateTiddlerWidget.tid (hunk 1 above)
Comment (pmario) [reply editions/tw5.com/tiddlers/widgets/ActionCreateTiddlerWidget.tid:23]: updated the link to v5.2.0  https://tiddlywiki.com/archive/full/TiddlyWiki-5.2.0.html#ActionCreateTiddlerWidget
